getPRs.go

```diff
@@ -0,0 +1,428 @@
+package main
```

Comment:
actually this is not the header.. you can take an example from minio server project.